packages/jest-circus/src/run.ts

```diff
@@ -117,8 +124,12 @@ const _runTest = async (test: Circus.TestEntry): Promise<void> => {
 
   await _callCircusTest(test, testContext);
```

Comment:
we should skip the test if `before` has failed, no?

Reply:
Yes - I think that makes a lot of sense. If the function call is skipped then the `test_fn_start` event is skipped which makes sense because the function won't be called anyways. Addressed in this [commit](https://github.com/facebook/jest/pull/10780/commits/eb0af7cd8f09e70eb00fd973a8748b064129d2de).

Do you think we should skip the `_runTest` function completely if there is an error during a `beforeAll` hook?
https://github.com/facebook/jest/blob/5e69254c8014a377342aaf596dab69d941a56f4e/packages/jest-circus/src/run.ts#L55-L57

Reply:
yeah I think so